source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -144,9 +166,28 @@ def _getNodeMathMl(self, node):
 
 	def _get_mathMl(self) -> str:
 		"""Return the MathML associated with a Formula tag"""
+		# There are two ways that MathML can be represented in a PDF:
+		# 1. As a series of nested tags, each with a MathML element as the value.
+		# 2. As a Formula tag with MathML as the value (comes from MathML in an Associated File)
 		if self.pdDomNode is None:
 			log.debugWarning("_get_mathMl: self.pdDomNode is None!")
 			raise LookupError
+
+		# see if it is MathML tagging is used
+		for childNum in range(self.pdDomNode.GetChildCount()):
+			try:
+				child = self.pdDomNode.GetChild(childNum).QueryInterface(IPDDomElement)
+			except COMError:
+				log.debugWarning(f"COMError trying to get childNum={childNum}")
```

Comment:
```suggestion
				log.debugWarning(f"COMError trying to get {childNum=}")
```

Reply:
I don't think your suggestion makes sense. You want an error message like "COMError trying to get childNum=1".